### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by cornelk**
* improves badges
* fixes line wrapping, the result is rendered the same
* fixes and extents the 
* removes old release notes
* fix CI by running `make regenerate`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -158,43 +180,14 @@ func NewBar(a, b Attrib) (*Bar, error) { ... }
...
-
-This issue will be properly resolved in a future release.
-
 ## Users
 
 These projects use gocc:
 
 * [gogo](https://github.com/shivansh/gogo) - [BNF file](https://github.com/shivansh/gogo/blob/master/src/lang.bnf) - a Go to MIPS compiler written in Go
 * [gonum/gonum](https://github.com/gonum/gonum) - [BNF file](https://github.com/gonum/gonum/blob/master/graph/formats/dot/internal/dot.bnf) - DOT decoder (part of the graph library of Gonum)
-* [llir/llvm](https://github.com/llir/llvm) - [BNF file](https://github.com/llir/llvm/blob/master/asm/internal/ll.bnf) - LLVM IR library in pure Go
```
Comment (awalterschulze) [inline README.md:194]: Maybe we can still link to the historical bnf

https://github.com/llir/llvm/blob/28149269dab73cc63915a9c2c6c7b25dbd4db027/asm/internal/ll.bnf

It is quite useful to have more examples when one is starting out

File: README.md [hunk 2]
```diff
@@ -147,7 +166,10 @@ Some action expression examples:
 << ast.TRUE, nil >>
 ```
 
-Contants, functions, etc. that are returned or called should be programmed by the user in his ast (Abstract Syntax Tree) package. The ast package requires that you define your own Attrib interface as shown above. All parameters passed to functions will be of this type.
+Constants, functions, etc. that are returned or called should be programmed by
```
Comment (awalterschulze) [inline README.md:169]: I don't know if these and most other newlines are necessary?

File: README.md [hunk 3]
```diff
@@ -1,8 +1,6 @@
-# New
-Have a look at [https://github.com/goccmack/gogll](https://github.com/goccmack/gogll) for scannerless GLL parser generation.
-# Gocc
-
-[![Build Status](https://travis-ci.org/goccmack/gocc.svg?branch=master)](https://travis-ci.org/goccmack/gocc)
+![Build Status](https://github.com/goccmack/gocc/workflows/build/badge.svg)
```
Comment (awalterschulze) [inline README.md:1]: Nice, very cool :D

File: README.md [hunk 4]
```diff
@@ -12,27 +10,39 @@ Gocc generates lexers and parsers or stand-alone DFAs or parsers from a BNF.
...
+
 ## Installation
 
 * First download and Install Go From http://golang.org/
 * Setup your GOPATH environment variable.
-* Next in your command line run: go get github.com/goccmack/gocc (go get will git clone gocc into GOPATH/src/github.com/goccmack/gocc and run go install)
-* Alternatively clone the source: https://github.com/goccmack/gocc . Followed by go install github.com/goccmack/gocc
-* Finally make sure that the bin folder where the gocc binary is located is in your PATH environment variable.
+* Next in your command line run: go get github.com/goccmack/gocc (go get will 
+  git clone gocc into GOPATH/src/github.com/goccmack/gocc and run go install)
```
Comment (awalterschulze) [inline README.md:37]: Good idea to make this verbatim :)

File: README.md (hunk 2 above)
Comment (cornelk) [reply README.md]: It makes the file easier to edit, the rendered result is the same

**Comment by cornelk** [comment]
@awalterschulze I updated the MR and description, sorry for the long delay :)

**Comment by awalterschulze** [comment]
Been waiting long enough for a second approver, I hope it is okay if I merge this.

**Pull request merged**